app/src/main/java/org/fossasia/openevent/general/ticket/TicketsFragment.kt

```diff
@@ -144,7 +145,9 @@ class TicketsFragment : Fragment() {
     override fun onDestroyView() {
         val activity = activity as? MainActivity
         activity?.supportActionBar?.setDisplayHomeAsUpEnabled(false)
+        invalidateOptionsMenu(activity)
         super.onDestroyView()
+
```

Comment:
Not required

Reply:
@pranavpandey1998official why is this marked resolved?